client/webserver/site/src/js/settings.ts

```diff
@@ -48,9 +48,7 @@ export default class SettingsPage extends BasePage {
     })
 
     Doc.bind(page.showPokes, 'click', () => {
-      const show = page.showPokes.checked || false
-      State.setCookie('popups', show ? '1' : '0')
-      app().showPopups = show
```

Comment:
Deleting this line seems wrong. The changes won't apply until refresh... 

Verified. Doesn't update without refresh.

Reply:
Phew, I got a bit confused by who/how is using `app().showPopups`, 

now I think I understand better how interfaces work in typescript, fixed.